datameta/api/openapi.yaml

```diff
@@ -15,7 +15,7 @@
 openapi: 3.0.0
 info:
   description: DataMeta
-  version: 0.21.0
+  version: 0.21.1
```

Comment:
```suggestion
  version: 0.22.0
```

Reply:
I thought since we did not add any endpoints, only changed the existing ones, we would only increase the 3rd digit